plugins/woocommerce/tests/e2e-pw/tests/merchant/order-search.spec.js

```diff
@@ -51,6 +50,25 @@ const queries = [
 	[ itemName, 'shipping item name' ],
 ];
 
+/**
+ * Check first if customer already exists. Delete if it does.
+ */
+const deleteCustomer = async ( api ) => {
```

Comment:
There are cases when this test customer failed to be deleted by earlier tests. This function ensures that the test will start without this pre-existing customer.